run.py

```diff
@@ -239,6 +242,20 @@ def keyboard_click(*KEYBOARD_NAMES):
 def grab_screen(n):
     return callback.get_screen_grab()
 
+@app.callback(
+    Output('calendar-screen','children'),
+    [Input('next-change_mth', 'n_clicks')])
+def next_month(n_clicks):
+    if n_clicks > 0:
+        change_next_month() 
```

Comment:
There's no `return` here, co I assume that this function return **None** to calendar-screen